### PR TITLE
[docs] Add deploy read replica clusters to the multi-dc section of Deploy

### PR DESCRIPTION
**Pull request by stevebang**
This PR add the steps for deploying a read replica cluster to the Multi-data center subsection of the Deploy guide.

### CODE REVIEW

File: docs/content/latest/deploy/multi-dc/_index.html [hunk 1]
```diff
@@ -21,27 +21,42 @@
...
   </div>
+
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+    <a class="section-link icon-offset" href="read-replica-clusters/">
+      <div class="head">
+        <img class="icon" src="/images/section_icons/explore/planet_scale.png" aria-hidden="true" />
+        <div class="title">Read replica clusters</div>
+      </div>
+      <div class="body">
+        Deploy a read replica cluster to asynchronously replicate data from a primary cluster.
```
Comment (schoudhury) [inline docs/content/latest/deploy/multi-dc/_index.html]: Should be "Deploy a read replica cluster to serve low-latency reads from a remote DC"

File: docs/content/latest/deploy/multi-dc/read-replica-clusters.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+showAsideToc: true
+---
+
+In a YugabyteDB deployment, replication of data between nodes of your primary cluster runs synchronously and guarantees strong consistency. Optionally, you can create a read replica cluster that asynchronously replicates data from the primary cluster and guarantees timeline consistency (with bounded staleness). A synchronously replicated primary cluster can accept writes to the system. Using a read replica cluster allows applications to serve low latency reads in remote regions.
+
+In a read replica cluster, read replicas are _observer nodes_ that do not participate in writes, but get a timeline-consistent copy of the data through asynchronous replication from the primary cluster.
+
+Use the steps below to deploy a read replica cluster using YugabyteDB. For information on deploying read replica clusters using the Yugabyte Platform, see [Read replicas](../../../manage/enterprise-edition/read-replicas/).
+
+## Deploy a read replicas cluster
```
Comment (schoudhury) [inline docs/content/latest/deploy/multi-dc/read-replica-clusters.md]: Should be "read replica cluster"

File: docs/content/latest/deploy/multi-dc/_index.html (hunk 1 above)
Comment (schoudhury) [inline docs/content/latest/deploy/multi-dc/_index.html]: Should be "Deploy a read replica cluster to serve low-latency reads from a remote DC"

File: docs/content/latest/deploy/multi-dc/read-replica-clusters.md (hunk 2 above)
Comment (stevebang) [reply docs/content/latest/deploy/multi-dc/read-replica-clusters.md]: Fixed.

**Pull request merged**